deeplake/core/vectorstore/test_deeplake_vectorstore.py

```diff
@@ -1769,3 +1770,8 @@ def test_uuid_fix(local_path):
     vector_store.add(text=texts, id=ids, embedding=embeddings, metadata=metadatas)
 
     assert vector_store.dataset.id.data()["value"] == list(map(str, ids))
+
+
+def test_delete_by_path_wrong_path():
```

Comment:
I know this is not directly related with this PR, but could you move this file under test directory? Just curious, without thie change in this PR, will the test pass?

Reply:
Tests for the vector store module has been added in these directories. If we are to move it, this file along with others should be moved, but in a different PR.

The test would fail without the changes.

Reply:
OK, we can structure the tests later. Just curious, for a path not exist, what's the original behaviour?